src/backend/src/controllers/description-Bullets.controllers.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from 'express';
+import { validationResult } from 'express-validator';
+
+export const checkWorkPackageDescriptionBullet = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({ errors: errors.array() });
+  }
+
+  const { body } = req;
+  const { submitterId, descriptionBulletId } = body;
```

Comment:
You have linting errors here and the B in the name of this file should be lowercase.